keras-dreambooth-sprint/README.md

```diff
@@ -135,20 +176,11 @@ tensorflow
 huggingface-hub
 ```
 
- And your app should be up and running!
-
-### Sprint **Prizes**
+And your app should be up and running!
 
-You can make submission in three themes: 
-
-- Nature and Animals (`nature`)
-- Sci-fi/Fantasy Universes (`sci-fi`)
-- Consentful (`consentful`): Partner up with an artist to fine-tune on their style, with their consent! Make sure to include a reference to the artist’s express consent (e.g. a tweet) in your model card.
-- Wild Card (`wild-card`): If your submission belongs to any category that is not above, feel free to tag it with wild-card so we can evaluate it out of that category.
-
-Add the category with their IDs to the model cards for submission and add `keras-dreambooth` to model card metadata in tags section. Here's an example [model card](https://huggingface.co/spaces/keras-dreambooth/example-submission/blob/main/README.md). All the submissions will be populated [in this leaderboard](https://huggingface.co/spaces/keras-dreambooth/leaderboard) and ranked according to likes on a given Space to determine the winners.
+## Sprint Prizes
```

Comment:
why did you remove categories? it's very ambiguous.

Reply:
I am in favor of keeping the categories.